zipkin-autoconfigure/collector-rabbitmq/src/main/java/zipkin/autoconfigure/collector/rabbitmq/ZipkinRabbitMQCollectorProperties.java

```diff
@@ -107,18 +112,31 @@ public void setUseSsl(Boolean useSsl) {
     this.useSsl = useSsl;
   }
 
+  public URI getUri() {
+    return uri;
+  }
+
+  public void setUri(URI uri) {
+    this.uri = uri;
+  }
+
   public RabbitMQCollector.Builder toBuilder()
-    throws KeyManagementException, NoSuchAlgorithmException {
+    throws KeyManagementException, NoSuchAlgorithmException, URISyntaxException {
```

Comment:
interesting that it throws so many things. It doesn't really matter though as we don't have api compatibility warranty on autoconfigure or zipkin-server types (though we should mention in a readme this hidden trivia)